prov/efa/src/rdm/rxr_env.h

```diff
@@ -34,6 +34,12 @@
 #ifndef _RXR_ENV_H
 #define _RXR_ENV_H
 
+/**
+ * Setting ibv_qp_attr.rnr_retry to this number when modifying qp
+ * to cause firmware to retry indefininetly.
```

Comment:
```suggestion
 * to cause firmware to retry indefinitely.
```

(I know this was copy/pasted as-is)

Reply:
Fixed in new revision.